### ISSUE TITLE
Command: migrate save Error:   No such table: mySchema._Migration

### ISSUE THREAD

**Issue by ZiNai** [prisma/migrate#578]
## Bug description
Prisma Version: 2.6.0
PostgreSQL: 12.x
Start from scratch (Prisma Migrate)
after set my Model, just run the command: prisma migrate save --experimental

SET the DEBUG=*, The full Error Output:
```

 MigrateEngine:stderr Sep 02 14:07:35.904  INFO migration_engine: Starting migration engine RPC server git_hash="650b5d0348ec38ae61e1e7db69bb54808418ede4" +0ms
  MigrateEngine:stderr Sep 02 14:07:36.131  INFO quaint::single: Starting a postgresql connection. +223ms
  MigrateEngine:rpc {
  MigrateEngine:rpc   jsonrpc: '2.0',
  MigrateEngine:rpc   error: {
  MigrateEngine:rpc     code: 4466,
  MigrateEngine:rpc     message: 'An error happened. Check the data field for details.',
  MigrateEngine:rpc     data: {
  MigrateEngine:rpc       is_panic: false,
  MigrateEngine:rpc       message: 'Failure during a migration command: Connector error. (error: Error querying the database: No such table: ercm._Migration\n' +
  MigrateEngine:rpc         '   0: migration_core::api::ListMigrations\n' +
  MigrateEngine:rpc         '             at migration-engine/core/src/api.rs:118)',
  MigrateEngine:rpc       backtrace: null
  MigrateEngine:rpc     }
  MigrateEngine:rpc   },
  MigrateEngine:rpc   id: 1
  MigrateEngine:rpc } +380ms

Error: Error: Failure during a migration command: Connector error. (error: Error querying the database: No such table: ercm._Migration
   0: migration_core::api::ListMigrations
             at migration-engine/core/src/api.rs:118)

```

## How to reproduce
Just from a sample .e.g [Start from scratch (Prisma Migrate)](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-prisma-migrate-typescript-postgres)

run command: prisma migrate save --experimental

## Expected behavior

I encountered such a situation when I upgraded to 2.6 today. The previous version did not happen

**Comment by Jolg42** [comment]
Looks like a duplicate of https://github.com/prisma/migrate/issues/572

**Comment by albertoperdomo** [comment]
Will be addressed in #572. Clsoing. Please follow that issue to track progress.
